instrumentation/cassandra/cassandra-3.0/src/test/groovy/CassandraClientTest.groovy

```diff
@@ -128,10 +122,42 @@ class CassandraClientTest extends AgentTestRunner {
     "SELECT * FROM users where name = 'alice' ALLOW FILTERING"                                         | "async_test"
   }
 
+  //TODO fix this test
+  @Ignore("""
```

Comment:
i'm getting this same failure in #1194 😭

it's _somehow_ due to fixing the logger name in `AgentTestRunner`, so if you revert that change you should be able to bring back this test, and i'll investigate as part of #1194